src/axom/core/examples/core_array_perf.cpp

```diff
@@ -0,0 +1,975 @@
+// Copyright (c) 2017-2023, Lawrence Livermore National Security, LLC and
+// other Axom Project Developers. See the top-level LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+/*! \file core_array_perf.cpp
+ *  \brief This example illustrates performance of array classes.
+ */
+
+// Axom includes
+#include "axom/core/StackArray.hpp"
+#include "axom/core/Array.hpp"
+#include "axom/core/MDMapping.hpp"
+#include "axom/core/execution/for_all.hpp"
+#include "axom/core/execution/nested_for_exec.hpp"
+#include "axom/core/execution/runtime_policy.hpp"
+#include "axom/core/memory_management.hpp"
+#include "axom/core/utilities/System.hpp"
+#include "axom/core/utilities/Timer.hpp"
+#include "axom/core/AnnotationMacros.hpp"
+
+#include "axom/fmt/format.h"
```

Comment:
Please include `fmt.hpp` instead of `fmt/format.h` -- it adds some necessary defines.

```suggestion
#include "axom/fmt.hpp"
```

Reply:
Done.